docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+---
+Order: 8
+Area: languages
+TOCTitle: Python
+ContentId: c2cb770d-571d-4edf-9eb9-b5b8977c21a0
+PageTitle: Python with Visual Studio Code
+DateApproved: 6/2/2016
+MetaDescription: Learn about Visual Studio Code features (syntax highlighting, snippets, linting) for Python development.
+---
+
+# Python on Visual Studio Code
+
+Python is fully supported in Visual Studio Code through an extension. Popular extensions in the [Marketplace](https://marketplace.visualstudio.com) provide code completion, linting, debugging, code formatting, snippets, and more. 
```

Comment:
Maybe link through an [extension](/docs/editor/extension-gallery.md)
in case readers aren't familiar at all with extensions.  We usually link to that introductory page when we first call out extensions in a topic.